edit portfolio.html

```diff
@@ -103,6 +103,9 @@ <h3>
         >
       </h3>
     </footer>
-    <script src="javascript.js"></script>
+    <script src="index.js"></script>
+    <script>
+      renderExperienceList();
+    </script>
```

Comment:
do we need to call this fucntion here explicitly like this? cant we call it when the page is loaded i.e by adding an eventListener for DOMContentLoaded in the javascript file ? like: 
`document.addEventListener("DOMContentLoaded", function () {
 yourFunction();
 }
 `